facturacion_mexico/facturacion_fiscal/timbrado_api.py

```
@@ -813,15 +813,15 @@ def _get_payment_form_for_invoice(self, sales_invoice) -> str:
-			# Extraer código SAT del formato "01 - Efectivo"
-			mode_parts = fiscal_doc.fm_forma_pago_timbrado.split(" - ")
-			if len(mode_parts) >= 2 and mode_parts[0].strip().isdigit():
-				return mode_parts[0].strip()
+			# Extraer código SAT — formato estándar: "01 Efectivo" (primeros 2 chars)
+			candidate = fiscal_doc.fm_forma_pago_timbrado[:2].strip()
+			if candidate.isdigit():
+				return candidate
 
 		# Prioridad 2: Lógica basada en método de pago SAT SOLO para PPD
 		if fiscal_doc.get("fm_payment_method_sat"):
 			if fiscal_doc.fm_payment_method_sat == "PPD":
```

Ruff (0.15.15)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)